LoginDemo/LearningSelenium/SingletonTest.cs

```diff
@@ -0,0 +1,101 @@
+﻿using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium;
+using LearningSelenium.Singletons;
+using LearningSelenium.Utilities;
+using OpenQA.Selenium.Support.UI;
+
+namespace LearningSelenium
+{
+    public class SingletonTest
+    {
+        WebDriver driver;
+        Utility utilityObj;
+
+        [SetUp]
+        public void Setup()
+        {
+            driver = new ChromeDriver();
+            //driver.Manage().Window.Maximize();
+            utilityObj = Utility.GetInstance(driver);
+            utilityObj.SetImplicitWait(20);
+
+        }
+
+        [Test]
+        public void TestLoginFail()
+        {
+            Singleton singletonObj = Singleton.GetInstance(driver);
+            driver.Navigate().GoToUrl("https://opensource-demo.orangehrmlive.com/web/index.php/auth/login");
+            utilityObj.SetImplicitWait(20);
+            singletonObj.EnterUserName("admin");
+            singletonObj.EnterPassword("admin1234");
+            singletonObj.LoginClick();
+            utilityObj.SetExplicitWait(singletonObj.GetErrorMessage(),20);
+            Assert.That(singletonObj.GetErrorMessage(),Is.EqualTo("Invalid credentials") , "username or password is incorrect");
```

Comment:
Better to use Assert.AreEqual

Reply:
I have used Assert.AreEqual but I got suggestion to change Asset.That from visual studio